playbooks/collect_activity/stats.py

```diff
@@ -0,0 +1,66 @@
+#!/edx/bin/python.edxapp
+# pylint: skip-file
+
+from ConfigParser import ConfigParser
+import gzip
+import os
+import re
+import sys
+
+import django
+
+
+nginx_access_pattern = re.compile(r'- - (?P<ipaddress>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})')
+log_path = '/edx/var/log/nginx'
+
+
+if __name__ == '__main__':
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'lms.envs.aws')
+    os.environ.setdefault('SERVICE_VARIANT', 'lms')
+    django.setup()
+
+    unique_hits = set()
+    files = sorted([
+        f for f in os.listdir(log_path)
+        if os.path.isfile(os.path.join(log_path, f)) and f.startswith('access.log')
+    ])
+
+    for file in files:
+        print >> sys.stderr, 'Parsing log file: {file}'.format(file=file)
+        file_path = os.path.join(log_path, file)
+        if file.endswith('.gz'):
+            handle = gzip.open(file_path, 'rb')
+        else:
+            handle = open(file_path, 'r')
+
+        for line in handle:
+            match = re.match(nginx_access_pattern, line)
+            if match:
+                unique_hits.add(match.group('ipaddress'))
+
+        handle.close()
+
+    from django.contrib.auth.models import User
+    from xmodule.modulestore.django import modulestore
+
+    ms = modulestore()
+
+    stats = {
+        'users': User.objects.count(),
+        'courses': len(ms.get_courses()),
+        'hits': len(unique_hits),
+    }
+
+    section_name = sys.argv[1]
```

Comment:
@bdero `section_name` is fairly generic. IIUC, this is the IP address of the host/app server; it would be worth using a variable name that reflects that.